File: 200-1000 Difficulty/Enough Space.py
#https://www.codechef.com/problems/ENSPACE?tab=statement
t=int(input())
for x in range(t):
    n,x,y=map(int,input().split())
    if(n-x-(2*y)>=0):
        print("YES")
    else:
        print("NO")

File: 2001000 Difficulty/Enough Space.cpp
//https://www.codechef.com/problems/ENSPACE
#include <iostream>
using namespace std;

int main() {
	int t,n,x,y;
	cin>>t;
	while(t--)
	{
	    cin>>n>>x>>y;
	    cout<<((x+(2*y))<=n?"YES":"NO")<<endl;
	}
	return 0;
}
